app/models/manageiq/providers/ansible_tower/shared/inventory/collector/target_collection.rb

```diff
@@ -0,0 +1,66 @@
+module ManageIQ::Providers::AnsibleTower::Shared::Inventory::Collector::TargetCollection
+  def connection
+    @connection ||= manager.connect
+  end
+
+  def initialize(_manager, _target)
+    super
+  end
```

Comment:
Why is this needed if all it does is `super` ?

Reply:
it's not, I'll remove it

Reply:
it's needed now :) it adds dependent targets